integration-tests/pom.xml

```diff
@@ -349,6 +349,19 @@
                 </includes-failsafe>
             </properties>
         </profile>
+        <profile>
```

Comment:
Can we remove the profile  crio-pipeline, as it is going to be replaced by wls-srg

Reply:
fixed